WcaOnRails/app/views/role_change_mailer/notify_role_end.erb

```diff
@@ -0,0 +1,10 @@
+<p>The role has been removed for <%= @role.user.name %> in <%= @group_type_name %> on <%= @role.end_date %>.</p>
```

Comment:
You say "The role", but which role is it?
I feel like in addition to the metadata below, we should also include some "general role title" instead of just silently assuming everybody knows which role we're talking about

Reply:
Current email is as below:
```
Subject: Role removed for Daniel in translators
Body:
The role has been removed for Daniel in translators on 20-01-2024.
The change has been executed by Gregor.
More details of the role were:
Locale: en
```

This already has enough details right? Should we add any more details?

Reply:
Yup, okay!